samples/templates/default-azuredeploy.json

```diff
@@ -264,6 +274,58 @@
                 ]
             }
         },
+        {
+            "condition": "[equals(parameters('solutionType'), 'FhirServerSqlServer')]",
+            "name": "[variables('serviceName')]",
+            "type": "Microsoft.Sql/servers",
+            "apiVersion": "2015-05-01-preview",
+            "location": "[resourceGroup().location]",
+            "tags": {
+                "FhirServerSolution": "[parameters('solutionType')]"
+            },
+            "properties": {
+                "administratorLogin": "[uniqueString(resourceGroup().id)]",
+                "administratorLoginPassword": "[parameters('sqlAdminPassword')]",
+                "version": "12.0"
+            },
+            "resources": [
+                {
+                    "condition": "[equals(parameters('solutionType'), 'FhirServerSqlServer')]",
+                    "apiVersion": "2017-10-01-preview",
+                    "dependsOn": [
+                        "[variables('serviceName')]"
+                    ],
+                    "location": "[resourceGroup().location]",
+                    "tags": {
+                        "FhirServerSolution": "[parameters('solutionType')]"
+                    },
+                    "name": "FHIR",
+                    "properties": {
+                        "collation": "SQL_Latin1_General_CP1_CI_AS"
+                    },
+                    "sku": {
+                        "name": "Standard",
```

Comment:
nit: should these be configurable through parameters?

Reply:
I think at some point, yes. There are 5 different settings for SKU and without knowing which configuration we were going to recommend, I went for hard coding the cheapest to get tests passing.

https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/2017-10-01-preview/servers/databases#Sku